.github/workflows/publish.yml

```diff
@@ -0,0 +1,67 @@
+on:
+  workflow_call:
```

Comment:
So we will not auto-publish, but it will require a manual intervention? That sounds better to me, I thought you wanted to automatically push every build.

Reply:
We could change it to workflow call but then we should discus when we plan to release new features. It may also help to use the stability info of features to make it user configurable which 'stream' they are on.